src/common/constants.ts

```diff
@@ -44,16 +44,15 @@ export const DEBUG_SERVER_SCRIPT_PATH = path.join(
 );
 
 /**
- * Path to the Python script that starts the native language server with an
- * appropriate `ruff` executable.
+ * Path to the Python script that tries to find the Ruff binary path.
  *
- * This should only be used as a fallback if the user has not specified either
- * the `path` setting or is not using the bundled import strategy.
+ * This should only be used as a fallback if the user has not specified the
+ * `path` setting or the import strategy isn't `useBundled`.
```

Comment:
```suggestion
 * This should only be used as a fallback if couldn't retrieve valid Ruff binary
 * from `path` setting or the import strategy isn't `useBundled`.
```
`path` setting could be specified but invalid items in it.

Reply:
Good catch. I'll update the documentation. Thanks!